### PR TITLE
ovirt: Fix pep8 issues

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

This PR fixes PEP8 and Python3 compatibility issues.

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

ovirt

##### ANSIBLE VERSION

```
2.5.0
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
shipit

**Review by mwperina — APPROVED** [review]
shipit

File: lib/ansible/modules/cloud/ovirt/ovirt_api_facts.py [hunk 1]
```diff
@@ -3,9 +3,6 @@
 # Copyright (c) 2017 Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-from __future__ import absolute_import, division, print_function
-__metaclass__ = type
-
```
Comment (abadger) [inline lib/ansible/modules/cloud/ovirt/ovirt_api_facts.py]: This boilerplate should remain.

**Comment by abadger** [comment]
I'm not sure I see that this PR actually fixes anything?  Which part of it is the Python3 fix?  What's the Python3 error that it is fixing?

If these files are in need of PEP8 changes right now, why aren't they presently in any of the PEP8 skip, ignore or legacy lists?

needs_info
needs_revision

**Comment by machacekondra** [comment]
Right, it isn't fixing any Python3 stuff, only pep8. I am unsure how pep8 is configured for ansible exactly, but it's fixing few pep8 issues like:

```bash
omachace ~/workspace/ansible/lib/ansible/modules/cloud/ovirt (devel) $ flake8 *.py
ovirt_cluster.py:532:9: F841 local variable 'sched_policy' is assigned to but never used
_ovirt_clusters.py:532:9: F841 local variable 'sched_policy' is assigned to but never used
ovirt_nics.py:272:1: E305 expected 2 blank lines after class or function definition, found 1
ovirt_permissions_facts.py:98:21: E126 continuation line over-indented for hanging indent
ovirt_permissions_facts.py:99:17: E126 continuation line over-indented for hanging indent
ovirt_permissions_facts.py:100:13: E123 closing bracket does not match indentation of opening bracket's line
ovirt_storage_domains.py:295:5: F401 'ovirtsdk4.types.HostStatus as hoststate' imported but unused
ovirt_templates.py:271:1: F401 'time' imported but unused
ovirt_templates.py:584:27: W503 line break before binary operator
ovirt_templates.py:585:27: W503 line break before binary operator
ovirt_vmpools_facts.py:106:1: E305 expected 2 blank lines after class or function definition, found 1
ovirt_vms.py:2079:27: W503 line break before binary operator
ovirt_vms.py:2080:27: W503 line break before binary operator
ovirt_vms.py:2081:27: W503 line break before binary operator
ovirt_vms.py:2082:27: W503 line break before binary operator
ovirt_vms.py:2083:27: W503 line break before binary operator
```

**Pull request closed without merging**